components/eamxx/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -168,12 +177,17 @@ yakl_process_target(scream_rrtmgp_yakl)
 # NOTE: cannot use 'PUBLIC' in target_link_libraries,
 #       since yakl_process_target already used it
 #       with the "plain" signature
+
+#<<<<<<< HEAD
+#find_library(NETCDF_C netcdf HINTS $ENV{NETCDF_C_PATH}/lib)
+#target_link_libraries(scream_rrtmgp_yakl ${NETCDF_C} rrtmgp scream_share)
+#=======
 find_library(NETCDF_C netcdf HINTS ${NetCDF_C_PATH}/lib)
```

Comment:
While at it, do you mind adding `${NetCDF_C_PATH}/lib64` to the hints? Since `lib64` is quite common, we should be proactive and add it to the list of hints (e..g, my laptop and workstation use lib64)

Reply:
thanks, done

Reply:
This code looks wrong. If we are building as part of a E3SM/CIME case, there's no need to find netcdf, it should already be a target. We can check if the `netcdf` target already exists or check if SCREAM_CIME_BUILD is on and not do this find if either is true.

Reply:
Note, this is issue is not related specifically to this PR.

Reply:
Probably, but the source won't build without netcdf. We only need to do this find if we are standalone eamxx.

Reply:
@jgfouca IMO, it's fine to call find_package (and the likes) regradless of the value of SCREAM_CIME_BUILD... When the pkg/lib/path has alreday been found, it should be a no-op.

That said, the warning reported by Naser is indeed wrong, as it's likning a path instead of a tgt/lib...

Reply:
Is it fine to mix find_package and find_library?